packages/edit-site/src/components/page-templates/style.scss

```diff
@@ -52,7 +52,13 @@
 }
 
 .page-templates-description {
-	white-space: normal;
+	display: block;
+	max-width: $maximum-text-width;
+	text-wrap: pretty;
```

Comment:
Since `text-wrap: pretty` doesn't seem to be applicable in Safari, I think `text-wrap:balance` is necessary as a fallback (#62233)